src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const orderedStyles = sourceString.split(';');
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy before each operation. The `split` method does not mutate the original string, so you can chain the necessary methods directly to the result of `split`. Consider chaining the `map` and `filter` methods to avoid using loops and to follow functional programming practices.